README.md

```diff
@@ -15,49 +16,63 @@ A python toolbox for working with the Ursa Platform APIs in ArcGIS Pro.
-  - ** All users must be "Invoice Only" customer type, if you are unsure what you are set to contact [support@ursaspace.com](mailto:support@ursaspace.com).
+  - \*\* All users must be "Invoice Only" customer type, if you are unsure what you are set to contact [support@ursaspace.com](mailto:support@ursaspace.com).
 - Admin privileges if ArcGIS Pro is installed at system level.
   - If you do not have Admin access, a clone of the default ArcGIS Pro Python Environment must be created in a directory that does not require Admin access. See [ESRI Documentation](https://pro.arcgis.com/en/pro-app/latest/arcpy/get-started/clone-an-environment.htm) for more details.
 - This python toolbox only uses packages available in the default python environment shipped with ArcGIS Pro.
 
 ## Downloading from GitHub
 
 - Use this link https://github.com/ursaspace/ursa-arcgis-toolbox/archive/refs/heads/main.zip
-- or "Download ZIP" - Find and select the green "< > Code" button, then select "Download ZIP" 
-<img width="1190" alt="download-zip" src="https://user-images.githubusercontent.com/123033437/215618368-4f7bdb8c-8e4a-4069-9794-edc4071e5b00.png">
+- or "Download ZIP" - Find and select the green "< > Code" button, then select "Download ZIP"
+  <img width="1190" alt="download-zip" src="https://user-images.githubusercontent.com/123033437/215618368-4f7bdb8c-8e4a-4069-9794-edc4071e5b00.png">
 
 ## Installation
-0) Unzip the compressed folder to your desired location
 
-1) In an ArcGIS Pro Document, open "Catalog"
+0. Unzip the compressed folder to your desired location
+
+1. In an ArcGIS Pro Document, open "Catalog"
 
-2) In Catalog, right-click "Toolboxes" and select "Add Toolbox"
-<img width="323" alt="Screenshot 2023-01-30 at 15 16 01" src="https://user-images.githubusercontent.com/123033437/215618434-a2fedc98-8153-42b7-b465-d901af682989.png">
+2. In Catalog, right-click "Toolboxes" and select "Add Toolbox"
+   <img width="323" alt="Screenshot 2023-01-30 at 15 16 01" src="https://user-images.githubusercontent.com/123033437/215618434-a2fedc98-8153-42b7-b465-d901af682989.png">
 
-3) Navigate to "UrsaPlatform.pyt" where you unzipped Ursa's Python Toolbox, then click "Ok"
+3. Navigate to "UrsaPlatform.pyt" where you unzipped Ursa's Python Toolbox, then click "Ok"
 
-4) Open the added toolbox. Listed are the available tools.
-<img width="323" alt="Screenshot 2023-01-30 at 15 16 21" src="https://user-images.githubusercontent.com/123033437/215618493-349f235f-2d0a-47cb-877e-e78138ab46e4.png">
+4. Open the added toolbox. Listed are the available tools.
+   <img width="323" alt="Screenshot 2023-01-30 at 15 16 21" src="https://user-images.githubusercontent.com/123033437/215618493-349f235f-2d0a-47cb-877e-e78138ab46e4.png">
 
 ## Usage
+
 Tools can be opened by double-clicking the item in the toolbox.
 
 ### Login
+
 - User must provide Ursa Platform credentials using the "Login" tool in order to use the Ursa Platform APIs.
```

Comment:
I would specify what happens if you try to use one of the order tools when you're not logged in - that way if anyone doesn't realize they didn't log in and gets the error we mention here, they'll easily be able to tell "oh, it's just that I didn't log in" and it'll be more clear that it's not some other sort of error

Reply:
Sure, added a line that if a user is not logged in, tools requiring authentication will fail.